project-base/tests/ShopBundle/Test/Codeception/Helper/LocalizationHelper.php

```diff
@@ -0,0 +1,253 @@
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Component\Money\Money;
+use Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory;
+use Shopsys\FrameworkBundle\Model\Localization\Localization;
+use Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade;
+use Shopsys\FrameworkBundle\Model\Product\Unit\UnitFacade;
+use Shopsys\FrameworkBundle\Twig\NumberFormatterExtension;
+use Tests\ShopBundle\Test\Codeception\Module\StrictWebDriver;
+
+class LocalizationHelper extends Module
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Localization\Localization
+     */
+    private $localization;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    private $domain;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Router\DomainRouterFactory
+     */
+    private $domainRouterFactory;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Pricing\Currency\CurrencyFacade
+     */
+    private $currencyFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\CurrencyFormatter\CurrencyFormatterFactory
+     */
+    private $currencyFormatterFactory;
+
+    /**
+     * @var \CommerceGuys\Intl\Currency\CurrencyRepositoryInterface
+     */
+    private $intlCurrencyRepository;
+
+    /**
+     * @var \Tests\ShopBundle\Test\Codeception\Module\StrictWebDriver
+     */
+    private $webDriver;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Model\Product\Unit\UnitFacade
+     */
+    private $unitFacade;
+
+    /**
+     * @var \Shopsys\FrameworkBundle\Twig\NumberFormatterExtension
+     */
+    private $numberFormatterExtension;
+
+    /**
+     * @param \Codeception\TestInterface $test
+     */
+    public function _before(TestInterface $test): void
+    {
+        /** @var \Tests\ShopBundle\Test\Codeception\Helper\SymfonyHelper $symfonyHelper */
+        $symfonyHelper = $this->getModule(SymfonyHelper::class);
+        /** @var \Tests\ShopBundle\Test\Codeception\Module\StrictWebDriver $strictWebDriver */
+        $strictWebDriver = $this->getModule(StrictWebDriver::class);
+        $this->webDriver = $strictWebDriver;
+        $this->localization = $symfonyHelper->grabServiceFromContainer(Localization::class);
+        $this->domain = $symfonyHelper->grabServiceFromContainer(Domain::class);
+        $this->domainRouterFactory = $symfonyHelper->grabServiceFromContainer(DomainRouterFactory::class);
+        $this->currencyFacade = $symfonyHelper->grabServiceFromContainer(CurrencyFacade::class);
+        $this->currencyFormatterFactory = $symfonyHelper->grabServiceFromContainer(CurrencyFormatterFactory::class);
+        $this->intlCurrencyRepository = $symfonyHelper->grabServiceFromContainer(CurrencyRepositoryInterface::class);
+        $this->unitFacade = $symfonyHelper->grabServiceFromContainer(UnitFacade::class);
+        $this->numberFormatterExtension = $symfonyHelper->grabServiceFromContainer(NumberFormatterExtension::class);
+    }
```

Comment:
Too many responsibilities! Please consider refactoring here.

Reply:
No, no, please :laughing: I do not want to consider that, the class serves it's purpose - the only responsibility is to help acceptance tests with localization, or not?

Reply:
I will try to extract some methods to new PriceHelper

Reply:
@s3tezsky extracted to new `NumberFormatHelper` class in f0421270c

Reply:
Good job! It looks better for me.

But to be honest the curency (facade, formatter, etc.) smells to me in `NumberFormatHelper`. Maybe we need the 3rd helper? I guess there will be hidden much digging so we can leave this one as is for now.